src/popup/components/NoOriginWarning.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <DetailsItem
+    :label="$t('pages.connectConfirm.unknownSource')"
+    warning
+    class="no-origin-warning"
+  >
+    <template #before-label>
+      <ExclamationPointCircle class="icon" />
+    </template>
+    <template #value>
+      <span>
```

Comment:
Using `span` around block elements is rather bad semantic. I suggest to use `div`.